packages/client/test/test-utils/fake/FakeEnvironment.ts

```diff
@@ -79,6 +87,10 @@ export class FakeEnvironment {
         return this.network
     }
 
+    getLogger(): FakeLogger {
+        return this.logger
+    }
+
```

Comment:
Should we provide a direct access to log entries here to make use easier?
```suggestion
    getLogEntries(): LogEntry[] {
        return this.logger.getEntries()
    }

```

Reply:
I'd keep the current style where each `FakeEnvironment` component is exposed as a getter, and the actual query/command methods are in that component. It is quite straightforward to assert `environment.getLogger().getEntries()` anyway.

This way we follow the "do only one thing, and do it well" principle in `FakeEnvironment`: the environment just manages the component, doesn't operate on them (and doesn't need to know what operations are available).